src/Mapping/Annotation/UploadableField.php

```diff
@@ -51,12 +52,26 @@ class UploadableField
      *
      * @throws \InvalidArgumentException
      */
-    public function __construct(array $options)
-    {
-        if (empty($options['mapping'])) {
+    public function __construct(
+        array $options = [],
+        string $mapping = null,
+        string $fileNameProperty = null,
+        string $size = null,
+        string $mimeType = null,
+        string $originalName = null,
+        array $dimensions = null
+    ) {
+        if (empty($options['mapping']) && empty($mapping)) {
             throw new \InvalidArgumentException('The "mapping" attribute of UploadableField is required.');
         }
 
+        $this->mapping = $mapping ?? $this->mapping;
```

Comment:
I don't understand why ` ?? $this->mapping` 🤔

Reply:
It's a mistake. My bad.

Reply:
No harm 🙂 but do you plan to submit a follow-up PR?

Reply:
OK, since my own (unrelated) PR has just been merged, I took the initiative to submit #1213 so that it can be "fixed" before the next release